prep-html-syntax/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Prep HTLM-Syntax</title>
+  <body>
```

Comment:
Be sure to close your `<head>` element before opening your `<body>` element.

Reply:
I've made the correction, thanks for catching that.